linera-storage/src/db_storage.rs

```diff
@@ -729,8 +727,7 @@ where
 impl<Store> DbStorage<Store, WallClock>
 where
     Store: KeyValueStore + Clone + Send + Sync + 'static,
-    Store::Error:
-        From<bcs::Error> + From<DatabaseConsistencyError> + Send + Sync + serde::ser::StdError,
+    Store::Error: Send + Sync + serde::ser::StdError,
```

Comment:
Not from this PR but why are we using `serde::ser::StdError`? (an alias `std::error::Error`)

Reply:
I know. I addressed this is in another branch (not yet PR) on the dual store. The answer is that it is not needed.
Let me do this cleanup in this PR.